change/@fluentui-react-button-a033d25e-6861-40b4-a52e-eb4632568acb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "#28576: [Bug]: Small <Button> is using 3px border radius",
```

Comment:
```suggestion
  "comment": "fix: Small Button now uses token for its border radius",
```

Reply:
Thanks for pointing out. It's now updated.